validator/remote/src/main/java/tech/pegasys/teku/validator/remote/RemoteBeaconNodeApi.java

```diff
@@ -79,6 +78,13 @@ public static BeaconNodeApi create(
     final RemoteValidatorApiChannel primaryValidatorApi =
         createRemoteValidatorApi(
             primaryEndpoint, okHttpClient, spec, preferSszBlockEncoding, asyncRunner);
+    final List<RemoteValidatorApiChannel> failoversValidatorApis =
```

Comment:
Not a fan of failovers. Maybe failoverValidatorApis would be enough? But don't feel fully entitled to judge :)